module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,17 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+std::deque<std::string> lengthSort(std::forward_list<std::string> list) {
```

Comment:
If you only read a list, than you can add **const** and **&** to type of argument.